upgrade-advisor.php

```diff
@@ -25,8 +25,15 @@ function isThere($setting) {
         }
     }
 
+    $googleapi_setttings = json_decode(get("https://maps.googleapis.com/maps/api/geocode/json?key=" .$google_maps_api_key. "&address=91409"));
```

Comment:
You could pull the setting as it's already defined.  https://github.com/radius314/yap/blob/master/functions.php#L6.